poc/vdaf_prio3.py

```diff
@@ -173,8 +188,12 @@ def shard_without_joint_rand(Prio3, meas, seeds):
             )
 
         # Generate the proof and shard it into proof shares.
-        prove_rand = Prio3.prove_rand(k_prove)
-        leader_proof_share = Prio3.Flp.prove(meas, prove_rand, [])
+        prove_rands = Prio3.prove_rands(k_prove)
+        leader_proof_share = []
```

Comment:
```suggestion
        leader_proofs_share = []
```

Reply:
I think this is reflected in the latest version.